lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -244,18 +228,18 @@ Ufe::SceneItem::Ptr UsdHierarchy::defaultParent() const
 #if UFE_PREVIEW_VERSION_NUM >= 2026
 Ufe::UndoableCommand::Ptr UsdHierarchy::reorderCmd(const Ufe::SceneItemList& orderedList) const
 {
-	std::vector<TfToken> orderedTokens;
+    std::vector<TfToken> orderedTokens;
 
-	for (const auto& item : orderedList) {
-	    orderedTokens.emplace_back(downcast(item)->prim().GetPath().GetNameToken());
-	}
+    for (const auto& item : orderedList) {
+        orderedTokens.emplace_back(downcast(item)->prim().GetPath().GetNameToken());
+    }
 
-	// create a reorder command and pass in the parent and its reordered children list
-	return UsdUndoReorderCommand::create(downcast(sceneItem())->prim(), orderedTokens);
+    // create a reorder command and pass in the parent and its reordered children list
+    return UsdUndoReorderCommand::create(downcast(sceneItem())->prim(), orderedTokens);
 }
 #endif
 
 #endif // UFE_V2_FEATURES_AVAILABLE
 
 } // namespace ufe
-} // namespace MayaUsd
+} // namespace MAYAUSD_NS_DEF
```

Comment:
This is a shame. The macro name is really not useful here.

Reply:
Clang-format auto-generates this comment and currently, we are giving it a macro. Something that may be newer versions of clang-format will handle better.